app/views/layouts/layout.slim

```diff
@@ -56,9 +56,23 @@ html lang="en"
                           div.nav-bar-icon-container
                             div.nav-bar-icon
                       ul.dropdown-menu id="mobileNavDropdown" style="display: none"
-                        -if @user.member?
-                          li.dropdown-header.filter-item Essentials
-                          ==render partial: "custom_links"
+                        -custom_links=render('custom_links')
+                        -if @user.member? || custom_links.length > 0
+                          li.dropdown-header.filter-item Quick Links
+                          -if custom_links.length > 0
```

Comment:
Not needed

Reply:
Same as above unless I'm not seeing something here